microcloud/cmd/microcloud/ask.go

```diff
@@ -599,6 +623,104 @@ func (c *CmdControl) askNetwork(sh *service.Handler, systems map[string]InitSyst
 		infos = append(infos, system.ServerInfo)
 	}
 
+	// Configure the Ceph networks.
+	lxdService := sh.Services[types.LXD].(*service.LXDService)
+	if lxdService == nil {
+		return fmt.Errorf("failed to get LXD service")
+	}
+
+	availableCephNetworkInterfaces, err := lxdService.GetCephInterfaces(context.Background(), bootstrap, infos)
+	if err != nil {
+		return err
+	}
+
+	if len(availableCephNetworkInterfaces) == 0 {
+		fmt.Println("No network interfaces found with IPs to set a dedicated Ceph network, skipping Ceph network setup")
+	} else {
+		if bootstrap {
+			microCloudInternalNetworkAddr := microCloudInternalSubnet.IP.Mask(microCloudInternalSubnet.Mask)
+			ones, _ := microCloudInternalSubnet.Mask.Size()
+			microCloudInternalNetworkAddrCIDR := fmt.Sprintf("%s/%d", microCloudInternalNetworkAddr.String(), ones)
+
+			publicCephSubnet, err := c.asker.AskString(fmt.Sprintf("What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph public traffic on? [default: %s] ", microCloudInternalNetworkAddrCIDR), microCloudInternalNetworkAddrCIDR, validate.IsNetwork)
+			if err != nil {
+				return err
+			}
+
+			internalCephSubnet, err := c.asker.AskString(fmt.Sprintf("What subnet (either IPv4 or IPv6 CIDR notation) would you like your Ceph internal traffic on? [default: %s] ", microCloudInternalNetworkAddrCIDR), microCloudInternalNetworkAddrCIDR, validate.IsNetwork)
```

Comment:
The default value here should be publicCephSubnet and not microCloudInternalNetworkAddrCIDR like we discussed the other day.

Reply:
fixed